inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/LogFileSource.java

```diff
@@ -247,7 +248,7 @@ private long getBytePositionByLine(long linePosition) throws IOException {
                 }
             }
         } catch (Exception e) {
-            LOGGER.error("getBytePositionByLine error {}", e.getMessage());
+            LOGGER.error("getBytePositionByLine error {}", e.getStackTrace());
```

Comment:
Please use ` LOGGER.error("getBytePositionByLine error :", e);`

Reply:
ok, fix it next issue